runner/src/client/sass/_global-styles.scss

```diff
@@ -0,0 +1,2 @@
+@import "../../../../node_modules/govuk-frontend/govuk/settings/global-styles";
+@import "../../../../node_modules/govuk-frontend/govuk/core/global-styles";
```

Comment:
I'm not sure anything here is getting compiled or used since it's not being imported by application.scss. We are also importing all of these in application.scss anyway in this line `@import "../node_modules/govuk-frontend/govuk/all";`